example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.7.10'
+    ext.kotlin_version = '1.8.0'
```

Comment:
Did we try to run it on lower flutter versions as well? 3.10.2 or 3.13.0?

Reply:
On lower  flutter version it can be run with kotlin v1.7.10, I guess newer flutter version forced to upgrade kotlin and  since kotlin version is defined in example app, customer have to define it in their `build.gradle` according to their flutter version

Reply:
On lower  flutter version it can be run with kotlin v1.7.10, I guess newer flutter version forced to upgrade kotlin and  since kotlin version is defined in example app, customer have to define it in their `build.gradle` according to their flutter version

Reply:
Sorted